File: src/main/java/your/group/name/projectname/ProjectName.java
package your.group.name.projectname;

import org.bukkit.plugin.java.JavaPlugin;

public class ProjectName extends JavaPlugin {
  @Override
  public void onEnable() {
      getLogger().info("Plugin enabled with version: " + this.getPluginMeta().getVersion());
  }

  @Override
  public void onDisable() {
  }
}

File: src/main/kotlin/your/group/name/projectname/ProjectName.kt
package your.group.name.projectname

import org.bukkit.plugin.java.JavaPlugin

class ProjectName : JavaPlugin()
{
  override fun onEnable()
  {
    logger.info("Plugin enabled with version: ${this.pluginMeta.version}")
  }
  
  override fun onDisable()
  {

  }

}
